llabake/Hello-Books
# Ft get book react redux 154038364

#### What does this PR do?
This PR  gets a single book
#### Description of Task to be completed?
Users should be able to view a book, upvote, post review, borrow and downvote it
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install all the dependency
* Run npm run dev
* visit the port where the webpack-dev-server started
* on successful login, on navigation to the allbooks page, on click of a book, you should be able to see the book and its details
#### What are the relevant pivotal tracker stories?
154038364
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/34731370-70b84546-f562-11e7-8d16-d053ff1f9f9d.png)
